.secrets.baseline

```diff
@@ -138,7 +138,7 @@
         "hashed_secret": "733c83df12b5f09020cfc0ad9411ba17e7d1a093",
         "is_secret": false,
         "is_verified": false,
-        "line_number": 3122,
+        "line_number": 3128,
```

Comment:
I recommend removing this from the PR so we can cherry-pick this fix.  Then you can fix the secrets baseline in a separate PR.